### PR TITLE
First swing at a CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by TimeDoctor**
Thanks to Timbo for the suggestion. This one is mostly based on the one
from Atom.

### CODE REVIEW

**Comment by zturtleman** [comment]
"Helping others" and the next two headers are missing a space between `###` and the header text. The space is required to display as a header.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# Contributing to The ioquake3 Project
+
+The following is a set of guidelines for contributing to ioquake3 which is hosted in the [The ioquake Group](https://github.com/ioquake) on Github. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
```
Comment (wtfbbqhax) [inline CONTRIBUTING.md]: Make it ##

**Comment by TimeDoctor** [comment]
Thanks, I've fixed those typos.

**Comment by timangus** [comment]
Looks good to me...

**Pull request merged**